api/HTMLElement.json

```diff
@@ -2254,10 +2254,10 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "3"
+              "version_added": "9"
```

Comment:
Was `oncopy` on some other interface before Firefox 9? How about the copy_event entry in Element.json, does that need updating? And is there a good reason it's over on `Element` instead of here?

Ditto for the following `oncut` and `onpaste`.

Reply:
We probably need to look into this a little bit deeper; it doesn't make sense that the event handlers are on HTMLElement, but the events themselves are on Element.

Reply:
Reverted these changes, will dive deeper into this another time.